### DISCUSSION TITLE
How to set the stabilizationWindowSeconds for scale down from N to 1

### DISCUSSION THREAD

**Discussion by thincal** [kedacore/http-add-on#880] [General]
It seems that scaledownPeriod is about the scale 1 to 0 related with the cooldownPeriod in KEDA, but what's the period about scaling from N to 1 in HPA ? Is that possible configured in HTTPScaledObject ? Thanks in advanced.

**Comment by linkvt** [comment]
Closing this discussion as I'm doing a cleanup right now, if this is a bug that still exists please create an issue, otherwise just let us know.
